### PR TITLE
Add copy about ordering exercises in exercises.md

### PR DESCRIPTION
**Pull request by jbergenson**
Proposing updated copy based on open issue #116.

### CODE REVIEW

File: language-tracks/configuration/exercises.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ We show the topics for an exercise on the website.
 This lets people optimize the learning experience to their own interests—skipping topics that they are not interested in, or that they already know a lot about, and doing deeper dives into topics that they're curious about.
 Take a look at the (non-exhaustive) [topics list][topics] for suggestions of topics to add.
 
+## Order of exercises
+
+Exercises that are not part of the core track (`"core": false`) should be added to `config.json` on a line between the last core exercise and the end of the file (i.e. all core exercises are located at the top of the file, above all side exercises).
```
Comment (SleeplessByte) [inline language-tracks/configuration/exercises.md]:
```suggestion
Exercises that are not part of the core track (`"core": false`) should be added to `config.json` below the last core exercise and the end of the file (i.e. all core exercises are located at the top of the file, above all side exercises).
```

**Review by SleeplessByte — APPROVED** [review]
I like. Check the suggestion tho :)

File: language-tracks/configuration/exercises.md (hunk 1 above)
Comment (kotp) [reply language-tracks/configuration/exercises.md]: Perhaps still a little bit confusing, how can it be both below the last core exercise and below the end of the file?  Should it be below the last core at the end of the file?

Comment (rpottsoh) [reply language-tracks/configuration/exercises.md]: Wouldn't it be enough to say after the last core exercise.  Do we really need to say before the end of the the file?

Comment (kotp) [reply language-tracks/configuration/exercises.md]: Right. I can put things (technically speaking) after the end of a file (post EOF character), but that doesn't mean I should.  Perhaps leave out mention of the end of the file altogether?

**Comment by jbergenson** [comment]
Made a mistake consolidating commits and set the branch back one commit too far. Going to close this request and submit a new request from scratch.

**Comment by kotp** [comment]
Don't worry about it, but it gives an opportunity and reason to research `git reflog`.

File: language-tracks/configuration/exercises.md (hunk 1 above)
Comment (SleeplessByte) [reply language-tracks/configuration/exercises.md]: Ha. My edit wasn't processed correctly. There's not meant to be a between anymore. This is the real one:

```suggestion
Exercises that are not part of the core track (`"core": false`) should be added to `config.json` below the last core exercise (i.e. all core exercises are located at the top of the file, above all side exercises).
```

**Pull request closed without merging**